modules/rostests/kmtests/ntos_mm/ZwMapViewOfSection.c

```diff
@@ -10,6 +10,23 @@
 #define IGNORE -99
 #define NEW_CONTENT "NewContent"
 #define NEW_CONTENT_LEN sizeof(NEW_CONTENT)
+#define IsInvalidParamStatus(Status) (        \
+    Status == STATUS_INVALID_PARAMETER     || \
+    Status == STATUS_INVALID_PARAMETER_MIX || \
+    Status == STATUS_INVALID_PARAMETER_1   || \
+    Status == STATUS_INVALID_PARAMETER_2   || \
+    Status == STATUS_INVALID_PARAMETER_3   || \
+    Status == STATUS_INVALID_PARAMETER_4   || \
+    Status == STATUS_INVALID_PARAMETER_5   || \
+    Status == STATUS_INVALID_PARAMETER_6   || \
+    Status == STATUS_INVALID_PARAMETER_7   || \
+    Status == STATUS_INVALID_PARAMETER_8   || \
+    Status == STATUS_INVALID_PARAMETER_9   || \
+    Status == STATUS_INVALID_PARAMETER_10  || \
+    Status == STATUS_INVALID_PARAMETER_11  || \
+    Status == STATUS_INVALID_PARAMETER_12     \
```

Comment:
Alternatively, for this range:
```suggestion
    (Status >= STATUS_INVALID_PARAMETER_1  && \
     Status <= STATUS_INVALID_PARAMETER_12)   \
```